internal/provider/resource_password_test.go

```diff
@@ -7,24 +7,25 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
```

Comment:
Could've saved some code churn via aliasing this to `fwresource` or something and keeping the existing `resource` import from the sdk 👍

Reply:
Good point. Fixed